src/ert/gui/tools/workflows/workflows_tool.py

```diff
@@ -8,7 +8,7 @@ class WorkflowsTool(Tool):
     def __init__(self, ert, notifier):
         self.notifier = notifier
         self.ert = ert
-        enabled = len(ert.resConfig().workflow_jobs) > 0
+        enabled = len(ert.resConfig().workflows.keys()) > 0
```

Comment:
I think this is the same thing
```suggestion
        enabled = len(ert.resConfig().workflows) > 0
```

Reply:
Seems like the proper thing to do. :+1: